### DISCUSSION TITLE
Understanding the modes

### DISCUSSION THREAD

**Discussion by xeruf** [complexlogic/rsgain#92] [Q&A]
Does custom mode assume all listed files are part of the same album? That is not quite clear from the manual.

Similarly for easy mode it does not mention whether it writes the tags to the files automatically.

**Comment by complexlogic** [comment]
>Does custom mode assume all listed files are part of the same album?

If you enable the album gain calculations with `-a`, yes. 

>Similarly for easy mode it does not mention whether it writes the tags to the files automatically.

Yes, the Easy Mode is set to tag mode 'i' by default, which writes tags.
